.gitignore

```diff
@@ -1,2 +1,2 @@
-#flux/oss-bel-migration/linkerd/enterprise-linkerd/
-*.DS_Store
\ No newline at end of file
+*.DS_Store
```

Comment:
Maybe .gitignore the .DS_Store directory.  It's a MacOS indexing thing, I think.

Reply:
Never mind.  Looks like you're doing that here.